lib/puppet/feature/base.rb

```diff
@@ -18,7 +18,7 @@
   begin
     # ruby
     require 'Win32API'          # case matters in this require!
-    require 'win32ole'
+    require 'win32ole' ; WIN32OLE.codepage = WIN32OLE::CP_UTF8
```

Comment:
Might be good to add a quick note here about this globally ensures we get strings with `Encoding::UTF_8` back across the board when using Ruby's late bound COM support through `WIN32OLE` (Don't need to mention it there, but our early-bound COM support goes through our FFI layer and we already handle string encodings properly there.)

Were you able to figure out how `WIN32OLE.locale` relates?  Might also be worth mentioning something here about that... and how the default is 2048 - which is not listed in the MS table at https://msdn.microsoft.com/en-us/library/ms912047(v=winembedded.10).aspx